templates/Dataland/v2/download-card.html.twig

```diff
@@ -33,6 +33,18 @@
                     Files
                 </div>
                 <div class="card-body">
+                    <div>
+                        {% for link in dataset.datasetSubmission.datasetLinks|default %}
+                            {% if(link.name == 'ERDDAP') %}
+                                    <button type="button" class="btn btn-primary btn-sm">
+                                        <img src="{{ asset('build/images/download-icon.svg') }}">
```

Comment:
if anyone has better svg-foo than me and can change this to white, please suggest. I didn't want to resort to a .png if I could avoid it.